### DISCUSSION TITLE
When will Support Go Modules in Imports ?

### DISCUSSION THREAD

**Discussion by Cylkal** [traefik/yaegi#1292] [Q&A] [answered]
I also encountered the problems summarized in https://github.com/traefik/yaegi/pull/1265. 
Like `import "go.uber.org/zap" error: unable to find source related to: "go.uber.org/zap"`, while import `"go.uber.org/zap"`.
Because we use go modules, and dependencies will not be stored in GOPATH, but in GOMODCACHE.

**Comment by ldez** [comment, marked as answer]
Hello,

you can follow issue #259 and pull request #1265.

**Reply by Cylkal** [reply]
Hope the sooner the better. Thank you very much.
